docs/source/bettertransformer/contribution/content.mdx

```diff
@@ -0,0 +1,176 @@
+
+## Models that should be supported
+
+In theory, any model that has a transformer encoder layer, similar to the classic encoder described in the ["Attention is all you need"](https://arxiv.org/abs/1706.03762) paper should be supported. 
+More specifically, a model that has an encoder block with a MultiHead-Attention module (with pre or post-attention layer norm) should be convertible to its `BetterTransformer` equivalent. The conditions can be summarized as follows:
+
+- Use classic Multi Head attention module (for example, [DeBERTa](https://arxiv.org/abs/2006.03654) cannot be supported)
+- Use either `gelu` or `relu` activation function   
+- Have an even number of attention heads
+- Do not use any attention bias (for eg `T5` uses attention bias, therefore cannot be supported)
+- `eps` must be equal between the first and second layer norms for each layer
+
+## How to convert a model into `BetterTransformer` format?
+
+### Step 1: Identifying the layer to change
+
+First, go to `optimum/bettertransformer/__init__.py` and you'll see the dictionary `BETTER_TRANFORMER_LAYERS_MAPPING_DICT`. This should contain the mapping between the Module that can be converted to its `BetterTransformer` equivalent.
+Let us try to do it step by step for `Bert`, first we need to identify the layers that needs to be replaced:
+```python
+>>> from transformers import AutoModel
+
+>>> model = AutoModel.from_pretrained("bert-base-uncased")
+>>> print(model)
+...
+          (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)
+          (dropout): Dropout(p=0.1, inplace=False)
+        )
+      )
+      (11): BertLayer(
+        (attention): BertAttention(
+          (self): BertSelfAttention(
+            (query): Linear(in_features=768, out_features=768, bias=True)
+            (key): Linear(in_features=768, out_features=768, bias=True)
+            (value): Linear(in_features=768, out_features=768, bias=True)
+            (dropout): Dropout(p=0.1, inplace=False)
+          )
+          (output): BertSelfOutput(
+            (dense): Linear(in_features=768, out_features=768, bias=True)
+            (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)
+            (dropout): Dropout(p=0.1, inplace=False)
+          )
+        )
+        (intermediate): BertIntermediate(
+          (dense): Linear(in_features=768, out_features=3072, bias=True)
+          (intermediate_act_fn): GELUActivation()
+        )
+        (output): BertOutput(
+          (dense): Linear(in_features=3072, out_features=768, bias=True)
+          (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)
+          (dropout): Dropout(p=0.1, inplace=False)
+        )
+      )
+    )
+  )
+  (pooler): BertPooler(
+    (dense): Linear(in_features=768, out_features=768, bias=True)
+    (activation): Tanh()
+  )
+)
+```
+You can clearly see that the layers that needs to be replaced are the `BertLayer` modules. 
```

Comment:
```suggestion
You can clearly see that the layers that needs to be replaced are the `BertLayer` modules since it contains the whole encoder layer module.
```